.github/workflows/nm-upload-assets.yml

```diff
@@ -1,4 +1,4 @@
-name: nm upload assets to GCP
+name: nm upload assets to GCP and pypi.org
```

Comment:
just "nm upload assets" ... i think our convention is just matching the name with the workflow filename ...